app/views/account/show.erb

```diff
@@ -7,6 +7,13 @@
     <p>View or reset your Exercism API key <a href="/account/key">here</a>.</p>
 
     <hr>
+    <% if profile.has_public_profile_key? %>
+      <p> Here's your <a href="<%= "/profiles/#{profile.username}/#{profile.user.public_profile_key}" %>"> url</a> for sharing your profile </p>
+    <% else %>
+      <p>Share your profile by clicking <a href="/account/set_public_profile_key">here</a>.</p>
+    <% end %>
```

Comment:
Let's clarify that the profile page on Exercism is only visible to other Exercism users by default, who can only see your solutions if they've done the exercises themselves.

Then we can explain that creating a "private share" url will let you share with specific people, and they will be able to see all of your solutions.